memcnn/models/tests/test_split_dim.py

```diff
@@ -0,0 +1,32 @@
+
+import pytest
+import torch
+
+from memcnn import AdditiveCoupling, AffineAdapterNaive, AffineCoupling, InvertibleModuleWrapper
```

Comment:
The InvertibleModuleWrapper is not needed for this test and removing it will avoid some CI tests failures. See below for more detailed rationale.
```suggestion
from memcnn import AdditiveCoupling, AffineAdapterNaive, AffineCoupling
```